Cargo.toml

```diff
@@ -1,28 +1,33 @@
 [package]
-name = "async-broadcast"
-version = "0.3.3"
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/smol-rs/async-broadcast"
-documentation = "https://docs.rs/async-broadcast"
-description = "Async broadcast channels"
-readme = "README.md"
-edition = "2018"
-keywords = []
-categories = []
 authors = [
     "Stjepan Glavina <stjepang@gmail.com>",
     "Yoshua Wuyts <yoshuawuyts@gmail.com>",
     "Zeeshan Ali <zeeshanak@gnome.org>",
 ]
+categories = []
+description = "Async broadcast channels"
+documentation = "https://docs.rs/async-broadcast"
+edition = "2018"
+keywords = []
+license = "MIT OR Apache-2.0"
+name = "async-broadcast"
+readme = "README.md"
+repository = "https://github.com/smol-rs/async-broadcast"
+version = "0.3.3"
 
 [features]
 
 [dependencies]
+easy-parallel = "3.1.0"
 event-listener = "2.5.1"
 futures-core = "0.3.14"
-easy-parallel = "3.1.0"
 
 [dev-dependencies]
+criterion = "0.3"
 doc-comment = "0.3.3"
 futures-lite = "1.11.3"
 futures-util = "0.3.14"
+
+[[bench]]
+harness = false
+name = "broadcast_bench"
```

Comment:
addition of benchmarking is awesome! However it's also a separate logical change so separate commit please.